### PR TITLE
Added support to view connected clients.

### PR DESCRIPTION
**Pull request by r-ef**
First PR ever, sorry in advance for crippled code :)

### CODE REVIEW

File: winerp/client.py [hunk 1]
```diff
@@ -110,6 +110,26 @@ async def __verify_client(self):
         await self.send_message(payload)
         logger.info("Verification request sent")
 
+    async def clients(self, timeout: int = 60):
```
Comment (AwesomeSam9523) [inline winerp/client.py:113]: Please add a doctring so it gets added to docs as well.

File: winerp/client.py [hunk 2]
```diff
@@ -110,6 +110,26 @@ async def __verify_client(self):
...
+        _uuid = str(uuid.uuid4())
+        payload = MessagePayload(
+            type=Payloads.response,
+            id=self.local_name,
+            uuid=_uuid,
+            destination="server"
```
Comment (AwesomeSam9523) [inline winerp/client.py:119]: Not good to name it as `server`, as someone can have a client with that name.
kindly change it to `winerp_server` instead.

File: winerp/server.py [hunk 3]
```diff
@@ -182,6 +182,11 @@ def __on_message(self, client, _, msg):
 
         if msg.type.response or msg.type.error or msg.type.function_call:
             logger.debug("Received Response Message from client %s" % client['address'][1])
+            if msg.destination == "server":
```
Comment (AwesomeSam9523) [inline winerp/server.py:185]: change to `winerp_server` here as well.

File: winerp/client.py (hunk 2 above)
Comment (AwesomeSam9523) [reply winerp/client.py:119]: Also would be good to not let people name their client as `winerp_server` when registering a client.

**Review by AwesomeSam9523 — CHANGES_REQUESTED** [review]

**Comment by nouman0103** [comment]
I think the best way would be to create a new payload type.

**Comment by AwesomeSam9523** [comment]
Yeah that can be done too.

**Pull request closed without merging**